src/encoded/static/components/search.js

```diff
@@ -235,16 +235,16 @@ class BiosampleComponent extends React.Component {
                             {`${separator}${lifeStage}${ageDisplay})`}
                         </a>
                         <div className="result-item__data-row">
```

Comment:
Why are you making these bold? I did not see the request in the ticket.

Reply:
They were already bold, I'm just making it consistent that we're using class names to make them bold instead of <strong>. You suggested doing that in one place in your first code review and I wanted to do it everywhere if we were doing it one place.

Reply:
Thanks for catching me on that :)